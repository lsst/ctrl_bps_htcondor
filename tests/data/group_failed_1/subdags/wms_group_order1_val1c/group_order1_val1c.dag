JOB label2_val1c_val2a "label2_val1c_val2a.sub" DIR "../../jobs/label2/val1c"
JOB label2_val1c_val2b "label2_val1c_val2b.sub" DIR "../../jobs/label2/val1c"
JOB label4_val1c_val2a "label4_val1c_val2a.sub" DIR "../../jobs/label4/val1c"
JOB label4_val1c_val2b "label4_val1c_val2b.sub" DIR "../../jobs/label4/val1c"
PARENT label2_val1c_val2a CHILD label4_val1c_val2a
PARENT label2_val1c_val2b CHILD label4_val1c_val2b
DOT group_order1_val1c.dot
NODE_STATUS_FILE group_order1_val1c.node_status
SET_JOB_ATTR bps_run_quanta= "label2:2;label4:2"
SET_JOB_ATTR bps_job_summary= "label4:2;label2:2"
