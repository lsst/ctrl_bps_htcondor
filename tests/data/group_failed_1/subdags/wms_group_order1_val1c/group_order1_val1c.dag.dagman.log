000 (10120.000.000) 2025-03-10 11:07:36 Job submitted from host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
    DAG Node: wms_group_order1_val1c
...
001 (10120.000.000) 2025-03-10 11:07:37 Job executing on host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
...
005 (10120.000.000) 2025-03-10 11:08:07 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
...
