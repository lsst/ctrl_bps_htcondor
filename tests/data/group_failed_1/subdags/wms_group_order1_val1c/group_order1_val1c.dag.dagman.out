03/10/25 11:07:37 ******************************************************
03/10/25 11:07:37 ** condor_scheduniv_exec.10120.0 (CONDOR_DAGMAN) STARTING UP
03/10/25 11:07:37 ** /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman
03/10/25 11:07:37 ** SubsystemInfo: name=DAGMAN type=DAGMAN(9) class=CLIENT(2)
03/10/25 11:07:37 ** Configuration: subsystem:DAGMAN local:<NONE> class:CLIENT
03/10/25 11:07:37 ** $CondorVersion: 24.0.4 2025-02-10 $
03/10/25 11:07:37 ** $CondorPlatform: X86_64-AlmaLinux_9.5 $
03/10/25 11:07:37 ** PID = 18630 RealUID = 27031
03/10/25 11:07:37 ** Log last touched time unavailable (No such file or directory)
03/10/25 11:07:37 ******************************************************
03/10/25 11:07:37 Using config source: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/etc/condor/condor_config
03/10/25 11:07:37 Using local config sources: 
03/10/25 11:07:37    /etc/condor/condor_config
03/10/25 11:07:37    /etc/condor/config.d/00-minicondor
03/10/25 11:07:37    /etc/condor/config.d/00-security
03/10/25 11:07:37    /etc/condor/config.d/10-stash-plugin.conf
03/10/25 11:07:37    /etc/condor/config.d/99-lsst
03/10/25 11:07:37    /etc/condor/condor_config.local
03/10/25 11:07:37    /home/testuser/.condor/user_config
03/10/25 11:07:37 config Macros = 99, Sorted = 99, StringBytes = 3220, TablesBytes = 3652
03/10/25 11:07:37 CLASSAD_CACHING is ENABLED
03/10/25 11:07:37 Daemon Log is logging: D_ALWAYS D_ERROR D_STATUS
03/10/25 11:07:37 DaemonCore: No command port requested.
03/10/25 11:07:37 DAGMAN_USE_STRICT setting: 1
03/10/25 11:07:37 DAGMAN_VERBOSITY setting: 3
03/10/25 11:07:37 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
03/10/25 11:07:37 DAGMAN_DEBUG_CACHE_ENABLE setting: False
03/10/25 11:07:37 DAGMAN_SUBMIT_DELAY setting: 0
03/10/25 11:07:37 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
03/10/25 11:07:37 DAGMAN_STARTUP_CYCLE_DETECT setting: False
03/10/25 11:07:37 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 100
03/10/25 11:07:37 DAGMAN_AGGRESSIVE_SUBMIT setting: False
03/10/25 11:07:37 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 1
03/10/25 11:07:37 DAGMAN_QUEUE_UPDATE_INTERVAL setting: 30
03/10/25 11:07:37 DAGMAN_DEFAULT_PRIORITY setting: 0
03/10/25 11:07:37 DAGMAN_SUPPRESS_NOTIFICATION setting: False
03/10/25 11:07:37 allow_events (DAGMAN_ALLOW_EVENTS) setting: 114
03/10/25 11:07:37 DAGMAN_RETRY_SUBMIT_FIRST setting: True
03/10/25 11:07:37 DAGMAN_RETRY_NODE_FIRST setting: False
03/10/25 11:07:37 DAGMAN_MAX_JOBS_IDLE setting: 1000
03/10/25 11:07:37 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
03/10/25 11:07:37 DAGMAN_MAX_PRE_SCRIPTS setting: 20
03/10/25 11:07:37 DAGMAN_MAX_POST_SCRIPTS setting: 20
03/10/25 11:07:37 DAGMAN_MAX_HOLD_SCRIPTS setting: 20
03/10/25 11:07:37 DAGMAN_MUNGE_NODE_NAMES setting: True
03/10/25 11:07:37 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
03/10/25 11:07:37 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
03/10/25 11:07:37 DAGMAN_ALWAYS_RUN_POST setting: False
03/10/25 11:07:37 DAGMAN_CONDOR_SUBMIT_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_submit
03/10/25 11:07:37 DAGMAN_CONDOR_RM_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_rm
03/10/25 11:07:37 DAGMAN_USE_DIRECT_SUBMIT setting: True
03/10/25 11:07:37 DAGMAN_PRODUCE_JOB_CREDENTIALS setting: True
03/10/25 11:07:37 DAGMAN_DEFAULT_APPEND_VARS setting: False
03/10/25 11:07:37 DAGMAN_INHERIT_ATTRS_PREFIX setting: 
03/10/25 11:07:37 DAGMAN_INHERIT_ATTRS setting: 
03/10/25 11:07:37 DAGMAN_ABORT_DUPLICATES setting: True
03/10/25 11:07:37 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
03/10/25 11:07:37 DAGMAN_PENDING_REPORT_INTERVAL setting: 60
03/10/25 11:07:37 DAGMAN_CHECK_QUEUE_INTERVAL setting: 28800
03/10/25 11:07:37 DAGMAN_AUTO_RESCUE setting: True
03/10/25 11:07:37 DAGMAN_MAX_RESCUE_NUM setting: 100
03/10/25 11:07:37 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
03/10/25 11:07:37 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
03/10/25 11:07:37 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
03/10/25 11:07:37 DAGMAN_MAX_JOB_HOLDS setting: 100
03/10/25 11:07:37 DAGMAN_HOLD_CLAIM_TIME setting: 20
03/10/25 11:07:37 ALL_DEBUG setting: D_FULLDEBUG
03/10/25 11:07:37 DAGMAN_SUPPRESS_JOB_LOGS setting: False
03/10/25 11:07:37 DAGMAN_REMOVE_NODE_JOBS setting: True
03/10/25 11:07:37 DAGMAN_REMOVE_JOBS_AFTER_LIMIT_CHANGE setting: False
03/10/25 11:07:37 DAGMAN will adjust edges after parsing
03/10/25 11:07:37 argv[0] == "condor_scheduniv_exec.10120.0"
03/10/25 11:07:37 argv[1] == "-Lockfile"
03/10/25 11:07:37 argv[2] == "group_order1_val1c.dag.lock"
03/10/25 11:07:37 argv[3] == "-Dag"
03/10/25 11:07:37 argv[4] == "group_order1_val1c.dag"
03/10/25 11:07:37 argv[5] == "-MaxIdle"
03/10/25 11:07:37 argv[6] == "1000"
03/10/25 11:07:37 argv[7] == "-CsdVersion"
03/10/25 11:07:37 argv[8] == "$CondorVersion: 24.0.4 2025-02-10 $"
03/10/25 11:07:37 argv[9] == "-dagman"
03/10/25 11:07:37 argv[10] == "/work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman"
03/10/25 11:07:37 argv[11] == "-AutoRescue"
03/10/25 11:07:37 argv[12] == "1"
03/10/25 11:07:37 argv[13] == "-DoRescueFrom"
03/10/25 11:07:37 argv[14] == "0"
03/10/25 11:07:37 argv[15] == "-do_recurse"
03/10/25 11:07:37 argv[16] == "-update_submit"
03/10/25 11:07:37 Workflow batch-id: <10093.0>
03/10/25 11:07:37 Workflow batch-name: <u_testuser_DM-46294_group_fail_20250310T160455Z.dag+10093>
03/10/25 11:07:37 Workflow accounting_group: <>
03/10/25 11:07:37 Workflow accounting_group_user: <>
03/10/25 11:07:37 DAGMAN_LOG_ON_NFS_IS_ERROR setting: False
03/10/25 11:07:37 Default node log file is: </work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1c/./group_order1_val1c.dag.nodes.log>
03/10/25 11:07:37 DAG Lockfile will be written to group_order1_val1c.dag.lock
03/10/25 11:07:37 DAG Input file is group_order1_val1c.dag
03/10/25 11:07:37 Parsing 1 dagfiles
03/10/25 11:07:37 Parsing group_order1_val1c.dag ...
03/10/25 11:07:37 Adjusting edges
03/10/25 11:07:37 Dag contains 4 total nodes
03/10/25 11:07:37 Bootstrapping...
03/10/25 11:07:37 Number of pre-completed nodes: 0
03/10/25 11:07:37 MultiLogFiles: truncating log file /work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1c/./group_order1_val1c.dag.nodes.log
03/10/25 11:07:37 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:37 Of 4 nodes total:
03/10/25 11:07:37  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:37   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:37     0       0        0       0       2          2        0        0
03/10/25 11:07:37 0 job proc(s) currently held
03/10/25 11:07:37 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeCount = 0.0; SubmitCycleTimeSum = 0.0; ]
03/10/25 11:07:37 Registering condor_event_timer...
03/10/25 11:07:38 Submitting HTCondor Node label2_val1c_val2a job(s)...
03/10/25 11:07:38 Submitting node label2_val1c_val2a from file label2_val1c_val2a.sub using direct job submission
03/10/25 11:07:38 	assigned HTCondor ID (10122.0.0)
03/10/25 11:07:38 Submitting HTCondor Node label2_val1c_val2b job(s)...
03/10/25 11:07:38 Submitting node label2_val1c_val2b from file label2_val1c_val2b.sub using direct job submission
03/10/25 11:07:38 	assigned HTCondor ID (10123.0.0)
03/10/25 11:07:38 Just submitted 2 jobs this cycle...
03/10/25 11:07:38 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:38 Of 4 nodes total:
03/10/25 11:07:38  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:38   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:38     0       0        2       0       0          2        0        0
03/10/25 11:07:38 0 job proc(s) currently held
03/10/25 11:07:38 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeAvg = 0.1104288101196289; SubmitCycleTimeCount = 1.0; SubmitCycleTimeMax = 0.1104288101196289; SubmitCycleTimeMin = 0.1104288101196289; SubmitCycleTimeStd = 0.1104288101196289; SubmitCycleTimeSum = 0.1104288101196289; ]
03/10/25 11:07:39 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:39 Reassigning the id of job label2_val1c_val2a from (10122.0.0) to (10122.0.0)
03/10/25 11:07:39 Event: ULOG_SUBMIT for HTCondor Node label2_val1c_val2a (10122.0.0) {03/10/25 11:07:38}
03/10/25 11:07:39 Number of idle job procs: 1
03/10/25 11:07:39 Reassigning the id of job label2_val1c_val2b from (10123.0.0) to (10123.0.0)
03/10/25 11:07:39 Event: ULOG_SUBMIT for HTCondor Node label2_val1c_val2b (10123.0.0) {03/10/25 11:07:38}
03/10/25 11:07:39 Number of idle job procs: 2
03/10/25 11:07:43 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:43 Event: ULOG_EXECUTE for HTCondor Node label2_val1c_val2a (10122.0.0) {03/10/25 11:07:42}
03/10/25 11:07:43 Number of idle job procs: 1
03/10/25 11:07:45 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:45 Event: ULOG_EXECUTE for HTCondor Node label2_val1c_val2b (10123.0.0) {03/10/25 11:07:44}
03/10/25 11:07:45 Number of idle job procs: 0
03/10/25 11:07:54 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:54 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1c_val2a (10122.0.0) {03/10/25 11:07:54}
03/10/25 11:07:54 Number of idle job procs: 0
03/10/25 11:07:54 Node label2_val1c_val2a job proc (10122.0.0) completed successfully.
03/10/25 11:07:54 Node label2_val1c_val2a job completed
03/10/25 11:07:54 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:54 Of 4 nodes total:
03/10/25 11:07:54  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:54   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:54     1       0        1       0       1          1        0        0
03/10/25 11:07:54 0 job proc(s) currently held
03/10/25 11:07:54 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.006981715559959412; EventCycleTimeCount = 16.0; EventCycleTimeMax = 0.1108040809631348; EventCycleTimeMin = 1.883506774902344E-05; EventCycleTimeStd = 0.02768606126672226; EventCycleTimeSum = 0.1117074489593506; LogProcessCycleTimeAvg = 0.0001823306083679199; LogProcessCycleTimeCount = 4.0; LogProcessCycleTimeMax = 0.0002160072326660156; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 3.314925773919051E-05; LogProcessCycleTimeSum = 0.0007293224334716797; SleepCycleTimeAvg = 1.001081094145775; SleepCycleTimeCount = 16.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001059055328369; SleepCycleTimeStd = 1.196312390635934E-05; SleepCycleTimeSum = 16.0172975063324; SubmitCycleTimeAvg = 0.006498224595013787; SubmitCycleTimeCount = 17.0; SubmitCycleTimeMax = 0.1104288101196289; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.02678229887645348; SubmitCycleTimeSum = 0.1104698181152344; ]
03/10/25 11:07:55 Submitting HTCondor Node label4_val1c_val2a job(s)...
03/10/25 11:07:55 Submitting node label4_val1c_val2a from file label4_val1c_val2a.sub using direct job submission
03/10/25 11:07:55 	assigned HTCondor ID (10124.0.0)
03/10/25 11:07:55 Just submitted 1 job this cycle...
03/10/25 11:07:55 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:55 Of 4 nodes total:
03/10/25 11:07:55  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:55   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:55     1       0        2       0       0          1        0        0
03/10/25 11:07:55 0 job proc(s) currently held
03/10/25 11:07:55 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.006592385909136604; EventCycleTimeCount = 17.0; EventCycleTimeMax = 0.1108040809631348; EventCycleTimeMin = 1.883506774902344E-05; EventCycleTimeStd = 0.02685493312214391; EventCycleTimeSum = 0.1120705604553223; LogProcessCycleTimeAvg = 0.0001823306083679199; LogProcessCycleTimeCount = 4.0; LogProcessCycleTimeMax = 0.0002160072326660156; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 3.314925773919051E-05; LogProcessCycleTimeSum = 0.0007293224334716797; SleepCycleTimeAvg = 1.001080260557287; SleepCycleTimeCount = 17.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001059055328369; SleepCycleTimeStd = 1.208239776414377E-05; SleepCycleTimeSum = 17.01836442947388; SubmitCycleTimeAvg = 0.02110550138685438; SubmitCycleTimeCount = 18.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.0671997287824521; SubmitCycleTimeSum = 0.3798990249633789; ]
03/10/25 11:07:56 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:56 Reassigning the id of job label4_val1c_val2a from (10124.0.0) to (10124.0.0)
03/10/25 11:07:56 Event: ULOG_SUBMIT for HTCondor Node label4_val1c_val2a (10124.0.0) {03/10/25 11:07:55}
03/10/25 11:07:56 Number of idle job procs: 1
03/10/25 11:07:56 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1c_val2b (10123.0.0) {03/10/25 11:07:55}
03/10/25 11:07:56 Number of idle job procs: 1
03/10/25 11:07:56 Node label2_val1c_val2b job proc (10123.0.0) completed successfully.
03/10/25 11:07:56 Node label2_val1c_val2b job completed
03/10/25 11:07:56 Event: ULOG_EXECUTE for HTCondor Node label4_val1c_val2a (10124.0.0) {03/10/25 11:07:56}
03/10/25 11:07:56 Number of idle job procs: 0
03/10/25 11:07:56 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:56 Of 4 nodes total:
03/10/25 11:07:56  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:56   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:56     2       0        1       0       1          0        0        0
03/10/25 11:07:56 0 job proc(s) currently held
03/10/25 11:07:56 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.02120780944824219; EventCycleTimeCount = 18.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.883506774902344E-05; EventCycleTimeStd = 0.06725886957801629; EventCycleTimeSum = 0.3817405700683594; LogProcessCycleTimeAvg = 0.0002028942108154297; LogProcessCycleTimeCount = 5.0; LogProcessCycleTimeMax = 0.0002851486206054688; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 5.420759812600637E-05; LogProcessCycleTimeSum = 0.001014471054077148; SleepCycleTimeAvg = 1.001081029574076; SleepCycleTimeCount = 18.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001059055328369; SleepCycleTimeStd = 1.216725100266827E-05; SleepCycleTimeSum = 18.01945853233337; SubmitCycleTimeAvg = 0.01999478591115851; SubmitCycleTimeCount = 19.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.06548561416076903; SubmitCycleTimeSum = 0.3799009323120117; ]
03/10/25 11:07:57 Submitting HTCondor Node label4_val1c_val2b job(s)...
03/10/25 11:07:57 Submitting node label4_val1c_val2b from file label4_val1c_val2b.sub using direct job submission
03/10/25 11:07:57 	assigned HTCondor ID (10125.0.0)
03/10/25 11:07:57 Just submitted 1 job this cycle...
03/10/25 11:07:57 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:57 Of 4 nodes total:
03/10/25 11:07:57  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:57   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:57     2       0        2       0       0          0        0        0
03/10/25 11:07:57 0 job proc(s) currently held
03/10/25 11:07:57 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.0201176593178197; EventCycleTimeCount = 19.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.883506774902344E-05; EventCycleTimeStd = 0.06553637084621217; EventCycleTimeSum = 0.3822355270385742; LogProcessCycleTimeAvg = 0.0002028942108154297; LogProcessCycleTimeCount = 5.0; LogProcessCycleTimeMax = 0.0002851486206054688; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 5.420759812600637E-05; LogProcessCycleTimeSum = 0.001014471054077148; SleepCycleTimeAvg = 1.001081240804572; SleepCycleTimeCount = 19.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001059055328369; SleepCycleTimeStd = 1.18602343372092E-05; SleepCycleTimeSum = 19.02054357528687; SubmitCycleTimeAvg = 0.02220709323883057; SubmitCycleTimeCount = 20.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.06450231220940159; SubmitCycleTimeSum = 0.4441418647766113; ]
03/10/25 11:07:58 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:58 Reassigning the id of job label4_val1c_val2b from (10125.0.0) to (10125.0.0)
03/10/25 11:07:58 Event: ULOG_SUBMIT for HTCondor Node label4_val1c_val2b (10125.0.0) {03/10/25 11:07:57}
03/10/25 11:07:58 Number of idle job procs: 1
03/10/25 11:07:58 Event: ULOG_EXECUTE for HTCondor Node label4_val1c_val2b (10125.0.0) {03/10/25 11:07:58}
03/10/25 11:07:58 Number of idle job procs: 0
03/10/25 11:08:05 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:08:05 Event: ULOG_JOB_TERMINATED for HTCondor Node label4_val1c_val2a (10124.0.0) {03/10/25 11:08:05}
03/10/25 11:08:05 Number of idle job procs: 0
03/10/25 11:08:05 Node label4_val1c_val2a job proc (10124.0.0) completed successfully.
03/10/25 11:08:05 Node label4_val1c_val2a job completed
03/10/25 11:08:05 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:08:05 Of 4 nodes total:
03/10/25 11:08:05  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:08:05   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:08:05     3       0        1       0       0          0        0        0
03/10/25 11:08:05 0 job proc(s) currently held
03/10/25 11:08:05 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01656631187156395; EventCycleTimeCount = 27.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.05607137663555104; EventCycleTimeSum = 0.4472904205322266; LogProcessCycleTimeAvg = 0.000242199216570173; LogProcessCycleTimeCount = 7.0; LogProcessCycleTimeMax = 0.0003898143768310547; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 8.530404732685415E-05; LogProcessCycleTimeSum = 0.001695394515991211; SleepCycleTimeAvg = 1.001080053823965; SleepCycleTimeCount = 27.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001059055328369; SleepCycleTimeStd = 1.250363912766849E-05; SleepCycleTimeSum = 27.02916145324707; SubmitCycleTimeAvg = 0.01586292471204485; SubmitCycleTimeCount = 28.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.05506488110287337; SubmitCycleTimeSum = 0.4441618919372559; ]
03/10/25 11:08:07 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:08:07 Event: ULOG_JOB_TERMINATED for HTCondor Node label4_val1c_val2b (10125.0.0) {03/10/25 11:08:07}
03/10/25 11:08:07 Number of idle job procs: 0
03/10/25 11:08:07 Node label4_val1c_val2b job proc (10125.0.0) completed successfully.
03/10/25 11:08:07 Node label4_val1c_val2b job completed
03/10/25 11:08:07 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:08:07 Of 4 nodes total:
03/10/25 11:08:07  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:08:07   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:08:07     4       0        0       0       0          0        0        0
03/10/25 11:08:07 0 job proc(s) currently held
03/10/25 11:08:07 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01544108884087924; EventCycleTimeCount = 29.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.05419533987589686; EventCycleTimeSum = 0.447791576385498; LogProcessCycleTimeAvg = 0.0002561807632446289; LogProcessCycleTimeCount = 8.0; LogProcessCycleTimeMax = 0.0003898143768310547; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 8.832389031045454E-05; LogProcessCycleTimeSum = 0.002049446105957031; SleepCycleTimeAvg = 1.001079707310117; SleepCycleTimeCount = 29.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001055955886841; SleepCycleTimeStd = 1.314679625031629E-05; SleepCycleTimeSum = 29.03131151199341; SubmitCycleTimeAvg = 0.01480563481648763; SubmitCycleTimeCount = 30.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.05328430982808501; SubmitCycleTimeSum = 0.4441690444946289; ]
03/10/25 11:08:07 All jobs Completed!
03/10/25 11:08:07 Note: 0 total node deferrals because of -MaxJobs limit (0)
03/10/25 11:08:07 Note: 0 total node deferrals because of -MaxIdle limit (1000)
03/10/25 11:08:07 Note: 0 total node deferrals because of node category throttles
03/10/25 11:08:07 Note: 0 total PRE script deferrals because of -MaxPre limit (20) or DEFER
03/10/25 11:08:07 Note: 0 total POST script deferrals because of -MaxPost limit (20) or DEFER
03/10/25 11:08:07 Note: 0 total HOLD script deferrals because of -MaxHold limit (20) or DEFER
03/10/25 11:08:07 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:08:07 Of 4 nodes total:
03/10/25 11:08:07  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:08:07   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:08:07     4       0        0       0       0          0        0        0
03/10/25 11:08:07 0 job proc(s) currently held
03/10/25 11:08:07 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01544108884087924; EventCycleTimeCount = 29.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.05419533987589686; EventCycleTimeSum = 0.447791576385498; LogProcessCycleTimeAvg = 0.0002561807632446289; LogProcessCycleTimeCount = 8.0; LogProcessCycleTimeMax = 0.0003898143768310547; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 8.832389031045454E-05; LogProcessCycleTimeSum = 0.002049446105957031; SleepCycleTimeAvg = 1.001079707310117; SleepCycleTimeCount = 29.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001055955886841; SleepCycleTimeStd = 1.314679625031629E-05; SleepCycleTimeSum = 29.03131151199341; SubmitCycleTimeAvg = 0.01480563481648763; SubmitCycleTimeCount = 30.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.05328430982808501; SubmitCycleTimeSum = 0.4441690444946289; ]
03/10/25 11:08:07 Wrote metrics file group_order1_val1c.dag.metrics.
03/10/25 11:08:07 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.01544108884087924; EventCycleTimeCount = 29.0; EventCycleTimeMax = 0.2696700096130371; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.05419533987589686; EventCycleTimeSum = 0.447791576385498; LogProcessCycleTimeAvg = 0.0002561807632446289; LogProcessCycleTimeCount = 8.0; LogProcessCycleTimeMax = 0.0003898143768310547; LogProcessCycleTimeMin = 0.0001440048217773438; LogProcessCycleTimeStd = 8.832389031045454E-05; LogProcessCycleTimeSum = 0.002049446105957031; SleepCycleTimeAvg = 1.001079707310117; SleepCycleTimeCount = 29.0; SleepCycleTimeMax = 1.001101016998291; SleepCycleTimeMin = 1.001055955886841; SleepCycleTimeStd = 1.314679625031629E-05; SleepCycleTimeSum = 29.03131151199341; SubmitCycleTimeAvg = 0.01480563481648763; SubmitCycleTimeCount = 30.0; SubmitCycleTimeMax = 0.2694292068481445; SubmitCycleTimeMin = 9.5367431640625E-07; SubmitCycleTimeStd = 0.05328430982808501; SubmitCycleTimeSum = 0.4441690444946289; ]
03/10/25 11:08:07 **** condor_scheduniv_exec.10120.0 (condor_DAGMAN) pid 18630 EXITING WITH STATUS 0
